### DISCUSSION TITLE
Copy to clipboard button for hmtl tables.

### DISCUSSION THREAD

**Discussion by wilcar** [quarto-dev/quarto-cli#4534] [Q&A] [answered]
Cand you add a Copy to clipboard button for hmtl tables ?
It will be great.

**Comment by mcanouil** [comment, marked as answer]
- HTML tables can be selected and copy/paste.
- Depending on the tools you are using to generate tables in code cells, some already has that feature.

Depending on what exactly is your use case, you could also just add it yourself as a Quarto extension, here the basic part: <https://www.w3schools.com/howto/howto_js_copy_clipboard.asp>.
